<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_save_curve">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>81</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>保存曲线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_data_collection">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>81</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="configuration">
    <property name="title">
     <string>配置</string>
    </property>
    <addaction name="communication_property_menu"/>
    <addaction name="curve_property_menu"/>
   </widget>
   <widget class="QMenu" name="about">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="about_software"/>
   </widget>
   <addaction name="configuration"/>
   <addaction name="about"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>configure</string>
   </property>
  </action>
  <action name="communication_property_menu">
   <property name="text">
    <string>通信设置</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="curve_property_menu">
   <property name="text">
    <string>数据采集设置</string>
   </property>
  </action>
  <action name="about_software">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
